Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -189,6 +189,7 @@ final class EffectCancellationTests: XCTestCase {
     XCTAssertEqual(values, [1])
   }
 
+  #if !os(Windows)
```

Comment:
l.204 makes less sense with this here :)
is it obvious why the test can't be made to pass on Windows?

Reply:
Tryna pin down why this test is failing! It’s not a flake since it seems to pass before this change ([manual run](https://github.com/thebrowsercompany/swift-composable-architecture/actions/runs/5662128707/job/15341457569) pointing at `feature/windows-explorations`)

Reply:
ahh I think I know what’s going on — since the merged chain is merged two at a time and erased, I bet `feature/windows-explorations` tried to raise the nesting ceiling by reaching into the inner publishers — so we probably need to find a `count` in that test that works for Windows (30 passed, and trying 50 now).

Reply:
niiice, [50 worked](https://github.com/thebrowsercompany/swift-composable-architecture/actions/runs/5662315867/job/15342124450?pr=45)